modules/nf-core/minimap2/align/main.nf

```diff
@@ -33,8 +33,8 @@ process MINIMAP2_ALIGN {
     minimap2 \\
         $args \\
         -t $task.cpus \\
-        "${reference ?: reads}" \\
-        "$reads" \\
```

Comment:
I thought that there was an issue with this before, not sure if that's still the case but is works fine with just $reference or $index in this case because I already perform the indexing as part of the prior indexing minimap2 module